test/utils.js

```diff
@@ -104,7 +105,13 @@ const fixtures = {
 		await db.query(
 			`UPDATE user_invites 
 			SET status = $1 WHERE email = $2`, [status, email] 
-		)}
+		)
+	},
+	 generateAccessToken (data){
+		const accessToken = jwt.sign( data , config("TOKEN_SECRET"),
+			{expiresIn: '24h'})
+		return accessToken
+	}
```

Comment:
export a parameterless function called `api` here in the utils and let it return the `request(app)` you're using in the `unlike-gif.e2e.test.js` 

so your tests will now be  `api().delete(...`  instead of `request(app).delete...`